src/sites/twitch-twilight/modules/directory/following.js

```diff
@@ -351,16 +376,21 @@ export default class Following extends SiteModule {
 	}
 
 	updateChannelCard(inst) {
-		//if (!this.isRouteAcceptable()) return;
-
 		this.parent.updateUptime(inst, 'props.viewerCount.createdAt', '.tw-card .tw-aspect > div');
 
+		
 		const container = this.fine.getHostNode(inst),
 			card = container && container.querySelector && container.querySelector('.tw-card');
-
+		
 		if ( container === null || card === null )
 			return;
-
+		
+		if (inst.props.streamType === 'watch_party') {
+			const hideVodcasts = this.settings.get('directory.hide-vodcasts');
+			if (hideVodcasts) container.parentElement.classList.add('hide');
+			else container.parentElement.classList.remove('hide');
```

Comment:
I prefer having if and else on their own lines for everything but a quick return at the start of a function.